apps/src/templates/lessonOverview/LessonOverview.jsx

```diff
@@ -110,6 +115,34 @@ class LessonOverview extends Component {
     return options;
   };
 
+  renderPrintOptions = () => {
+    const pdfDropdownOptions = this.compilePdfDropdownOptions();
+
+    if (pdfDropdownOptions.length > 0 && userLocale === 'en-US') {
+      return pdfDropdownOptions.map(option => (
+        <a
```

Comment:
these act like buttons but are `a` tags... is that a limitation of this component?

Reply:
I just kept the same tags so the style was consistent across users without having to dig too much into the styles.

Reply:
Actually, it does not matter whether we pass links or buttons, the parent component re-tags everything as links:
https://github.com/code-dot-org/code-dot-org/blob/5d7b3eed7c8bbf024d7764919002f65afb93ccf1/apps/src/templates/DropdownButton.js#L105-L120